libultraship/libultraship/ImGuiImpl.cpp

```diff
@@ -1465,7 +1558,7 @@ namespace SohImGui {
                         ImGui::PushStyleVar(ImGuiStyleVar_Alpha, ImGui::GetStyle().Alpha * 0.5f);
                     }
 
-                    EnhancementCheckbox("Enable Beta Quest", "gEnableBetaQuest");
+                    PaddedEnhancementCheckbox("Enable Beta Quest", "gEnableBetaQuest", true, false);
                     Tooltip("Turns on OoT Beta Quest. *WARNING* This will reset your game.");
                     betaQuestEnabled = CVar_GetS32("gEnableBetaQuest", 0);
                     if (betaQuestEnabled) {
```

Comment:
could you remove
```
ImGui::Text("After Slider Beta Quest World: %d", betaQuestWorld);
```
and 
```
ImGui::Text("After Clamp Beta Quest World: %d", betaQuestWorld);
```

they were left in on accident and i figure just removing them with this PR will be easier than dealing with a merge conflict

Reply:
There's more merge conflicts than just that, so I'll just try and get the latest zhora merged into this instead.

Reply:
yes, but that hasn't been updated in latest zhora, the idea was to hold off on adding more to your imgui merge plate by waiting for your PR to fix that (but if you can just get it added that'd be perfect)

Reply:
You probably don't need to handle this here, but could be low hanging fruit #1103

Reply:
> You probably don't need to handle this here, but could be low hanging fruit #1103

I'd prefer to pick this up in a seperate PR if neccesary. I just want this to be in sooner rather than later because merge conflicts are getting really hard to sift through.

Reply:
@briaguya-ai Text is removed in latest commit.